Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -102,23 +188,42 @@ private IPackageManager CreatePackageManager(string rootDir)
             return mgr;
         }
 
-        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath)
+        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath, params IPackage[] dependencies)
```

Comment:
Refactored to more closely resemble the CreatePackage method in AnalyzerPluginGeneratorTests, mainly so that we can have easily add dependencies for the above integration tests.